docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/cinema_service
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py migrate &&
+             python manage.py runserver 0.0.0.0:8000"
```

Comment:
The `command` section chains multiple commands using `&&`. If any command fails, it will stop the execution of subsequent commands. Ensure that each command is robust and handles errors appropriately.